test/prevent-abbreviations.js

```diff
@@ -200,64 +228,62 @@ ruleTester.run('prevent-abbreviations', rule, {
 	invalid: [
 		noFixingTestCase({
 			code: 'let e',
-			errors: createErrors('Please rename the variable `e`. Suggested names are: `error`, `event`. A more descriptive name will do too.')
+			errors: createErrors(formatMessage('e', ['event', 'error'], 'variable'))
```

Comment:
Let's keep the old `errors` value here as per this comment https://github.com/sindresorhus/eslint-plugin-unicorn/pull/337/files#r300863993

Reply:
restored